### ISSUE TITLE
[BUG] <BUG> `plugins.treesitter.folding = true;` causes extremely slow startup

### ISSUE THREAD

**Issue by martiege** [nix-community/nixvim#2223]
| Field        | Description |
|--------------|-------------|
| Plugin       | `treesitter`  | 
| Nixpkgs      | `unstable` | 
| Home Manager | `unstable` | 

- [x] I have read the [FAQ](https://nix-community.github.io/nixvim/user-guide/faq.html) and my bug is not listed there.

## Description

Setting `plugins.treesitter.folding = true;` changes my startup time from ~160 ms to ~9 s. 

Specifically it seems like `opening buffers` and `VimEnter autocommands` takes about 4.5 s each... This consistently happens when I toggle the `plugins.treesitter.folding` attribute. 

## Minimal, Reproducible Example (MRE)

```nix
programs.nixvim = {
}
```

**Comment by saygo-png** [comment]
Does this still happen? I can't replicate it.

**Comment by martiege** [comment]
No, it seems to have been fixed during some update. Not sure which, so I'll close the issue.
